src/nvim/fileio.c

```diff
@@ -6615,7 +6616,7 @@ void aucmd_restbuf(aco_save_T *aco)
       close_tabpage(curtab);
     }
 
-    restore_snapshot(SNAP_AUCMD_IDX, false);
+    /* restore_snapshot(SNAP_AUCMD_IDX, false); */
     (void)win_comp_pos();       // recompute window positions
```

Comment:
this should also be removed I think (as window layout did not change)

Reply:
Yup, already removed in my local copy!